src/lib/sys.h

```diff
@@ -203,6 +203,12 @@ int sys_rm(const char *filename);
  */
 #define systemctl_cmd(...) run_command_quiet(SYSTEMCTL, __VA_ARGS__)
 
+/**
+ * @brief Run a systemctl command with the informed parameters in a path
+ */
+#define systemctl_cmd_path(path, ...) \
+	path ? run_command_quiet(SYSTEMCTL, path, __VA_ARGS__) : run_command_quiet(SYSTEMCTL, __VA_ARGS__)
```

Comment:
Instead of receiving a parameter "--root=my_path", we should receive the path here. And call it

    run_command_quiet(SYSTEMCTL, "--root", path, __VA_ARGS__)

It's a better API to have.

Reply:
Would that work... because systemctl man pages expects its as --root=path

Reply:
Usually `--param=val` is the same as `--param val`

Reply:
Tested works! Thanks i didnt know :)